r/examples/example_rsparkling.R

```diff
@@ -15,13 +15,17 @@
 # devtools::install_github("h2oai/sparkling-water", subdir = "/r/rsparkling")
 # devtools::install_github("rstudio/sparklyr")
 
+#Set version of spark. 
+#Note: At the time of spark_connect sparklyr will call the spark_dependencies function in the rsparkling package.
+options(rsparkling.sparklingwater.version = "1.6.7")
```

Comment:
I actually like the `spark_connect(master = ..., extensions = add(rsparkling(version="1.6.8")) )` more than the `options` one, but I got the feeling that it wasn't possible to do, or am I wrong @navdeep-G ?